docs/source/release/v6.4.0/mantidworkbench.rst

```diff
@@ -5,31 +5,26 @@ Mantid Workbench Changes
 .. contents:: Table of Contents
    :local:
 
-New and Improved
-----------------
-- New general setting: `Provide auto-completion suggestions in the script editor` for making auto-completion optional in the script editor
+New Features
+------------
+
 
 Bugfixes
 --------
```

Comment:
Do these two want to be under a general heading like it's formatted in `direct_geometry.rst` since they're currently at the same level as the `Intrument Viewer` Heading?

Reply:
We could do that. This has been based on previous release notes and historically Direct have had a General section whereas Workbench went straight in with New Features etc. 

What it will mean is adding a new folder to the file structure, moving some folders and files around and updating the documentation with the new structure. Happy to do it but we need to be certain we definitely want to do this. 

@martyngigg - any thoughts?

Reply:
I think in this case everything is "general" as it's workbench and should apply to everyone so no extra heading is needed. The top level being new/bug fix reads as this applies to workbench and the separate sections later can then make changes in the other tools more obvious. I'm happy with how this looks now.